app/views/pages/download.html.erb

```diff
@@ -6,10 +6,10 @@
     RubyGems is a package management framework for Ruby. Download the latest version here:
   </p>
   <div id="formats" class="download__formats">
-    <%= link_to "tgz", "http://production.cf.rubygems.org/rubygems/rubygems-#{version_number}.tgz", class: "download__format" %>
-    <%= link_to "zip", "http://production.cf.rubygems.org/rubygems/rubygems-#{version_number}.zip", class: "download__format" %>
-    <%= link_to "gem", "http://production.cf.rubygems.org/rubygems/rubygems-update-#{version_number}.gem", class: "download__format" %>
-    <%= link_to "git", "http://github.com/rubygems/rubygems", class: "download__format" %>
+    <%= link_to "tgz", "https://rubygems.global.ssl.fastly.net/rubygems/rubygems-#{version_number}.tgz", class: "download__format" %>
+    <%= link_to "zip", "https://rubygems.global.ssl.fastly.net/rubygems/rubygems-#{version_number}.zip", class: "download__format" %>
+    <%= link_to "gem", "https://rubygems.global.ssl.fastly.net/rubygems/rubygems-update-#{version_number}.gem", class: "download__format" %>
```

Comment:
@reedloden Can you replace these domains with `https://rubygems.org/`. I've added redirects to our current CDN links.